Tools/SandboxTest.ps1

```diff
@@ -0,0 +1,125 @@
+# Parse Arguments
+
+Param(
+  [Parameter(Mandatory, HelpMessage = "The path for the Manifest.")]
+  [String] $Manifest
+)
+
+if (-not (Test-Path -Path $Manifest -PathType Leaf)) {
+  throw 'The Manifest file does not exist.'
+}
+
+# Validate manifest file
+# We can't rely on status code until https://github.com/microsoft/winget-cli/issues/312 is solved
+$validationResult = winget.exe validate $Manifest
+if ($validationResult -like '*Manifest validation failed.*') {
+  throw 'Manifest validation failed.'
+}
+
+# Check if Windows Sandbox is enabled
+
+if (-Not (Test-Path "$env:windir\System32\WindowsSandbox.exe")) {
+  Write-Error -Category NotInstalled -Message @'
+Windows Sandbox does not seem to be available. Check the following URL for prerequisites and further details:    
+https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-sandbox/windows-sandbox-overview
+  
+You can run the following command in an elevated PowerShell for enabling Windows Sandbox:
+Enable-WindowsOptionalFeature -Online -FeatureName 'Containers-DisposableClientVM'
+'@ -ErrorAction Stop
+}
+
+# Set dependencies
+
+$desktopAppInstaller = @{
+  fileName = 'Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.appxbundle'
+  url      = 'https://github.com/microsoft/winget-cli/releases/download/v0.1.4331-preview/Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.appxbundle'
+  hash     = 'e1b5aa89c7354dd39db38c2ac33f0455c4529eb4cf77f023028d955122ea8377'
+}
+
+$vcLibs = @{
+  fileName = 'Microsoft.VCLibs.140.00_14.0.27810.0_x64__8wekyb3d8bbwe.Appx'
+  url      = 'https://github.com/felipecassiors/winget-pkgs/raw/da8548d90369eb8f69a4738dc1474caaffb58e12/Tools/SandboxTest_Temp/Microsoft.VCLibs.140.00_14.0.27810.0_x64__8wekyb3d8bbwe.Appx'

```

Comment:
We should make sure dependencies are coming from an official source. If you deleted your fork or make alterations in the future, this could fail start to fail.

Reply:
For sure! As I said I'm just using it as a placeholder.

Reply:
I'll mark this conversation as closed, because we have another open for the same matter.